app/scss/style.scss

```diff
@@ -1,6 +1,153 @@
 @import 'imports/mixins';
 @import 'imports/variables';
+@import 'imports/listingPage';
+@import 'imports/salesNavbar';
+@import 'imports/jumbotron';
+@import 'imports/bootstrapfilters';
+
+@font-face {
+  font-family:'Merriweather', serif;
+  src: url("../../resources/Merriweather-Regular.ttf");
+}
+
+* {
+  box-sizing: border-box
+}
+
+
 body {
-  background: red;
+  background: $tertiary;
   font-size: 10px;
+  margin: 0;
+  color: #E7ECF8;
+}
+
+button:focus {
+  tab-index: -1;
+}
+
+.basicNavbar {
+  background: #161B31;
+  height: 50px;
+}
+
+.basicNavbarLinks {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr;
+  justify-items: center;
+  align-items: center;
+  padding: 5px 0;
+  margin: 5px;
+  width: 100%;
+  height: 40px;
+  background-color: #21264D;
+
+  a:hover, a:active, a:visited {
+    color: inherit;
+    text-decoration: none;
+  }
+}
+
+.navLinks {
+  font-size: 1.5rem;
+  margin: 0;
+  font-family: Verdana, sans-serif;
+  color: #E7ECF8;
+  border: 0;
+  background-color: #21264D;
+  justify-content: space-around;
+}
+
+.jumbotron {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 400px;
+  position: relative;
+  background-image: url("../../resources/builderimg2.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+    h1 {
+      margin-top: 0;
+      margin-bottom: 0;
+      font-size: 6rem;
+    }
+    h2 {
+      margin-top: 0;
+      margin-bottom: 0;
+      font-size: 2.5rem;
+    }
+
+}
+
+.smallTitle {
+  display: none;
+}
+
+.overlay {
+  font-family: 'Merriweather', serif;
+  color: #E7ECF8;
+  width: 100%;
+  height: 400px;
+  font-size: 3rem;
+  text-align: center;
+  opacity: 100%;
+  margin-top: 0;
+  margin-bottom: auto;
+  padding-top: 5%;
+  padding-bottom: 5%;
+}
+
+@media screen and (max-width:710px){
+  .jumbotron {
+    height: 330px;
+    overflow: hidden;
+  }}
+
+.boostrapNavbar {
+  background: #21264D;
+  display: flex;
+  justify-content: space-around;
+  height: 50px;
+  color: $primary;
+}
+
+.pageContent {
+  padding-top: 20px;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  margin: auto auto;
+}
+
+@media screen and (max-width:900px) {
+  .pageContent {
+    display:grid;
+    grid-template-columns: 1fr 1fr;
+    margin:auto;
+  }
+  .listingCard {
+    justify-content: space-around;
+  }
+}
+
+@media screen and (max-width:450px) {
+  .pageContent {
+    display: grid;
+    grid-template-columns: 1fr;
+    margin: auto;
+  }
+  .listingCard {
+    width:350px;
+  }
+  .jumbotron {
+    display:none;
+  }
+  .smallTitle {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color:  #E7ECF8;
+    position: relative;
+  }
```

Comment:
Can any of these css chunks go into the imports folder? Atleast the .jumbotron and media query should go into their respective files